editor/editor_node.h

```diff
@@ -465,6 +466,12 @@ class EditorNode : public Node {
 	void _vp_resized();
 
 	void _save_scene(String p_file, int idx = -1);
+	void _discard_changes();
+	void _discard_changes(const String &p_str) {
```

Comment:
I think it's a little bit weird to have two methods with the same name. I know why you did it and what it does, but it could be a little bit more explicit that this one is for the connection to the signal and the other one is the one that does the work. Maybe a comment or a rename.

But I could be wrong, other UI gurus should take a look anyway 😄